tests/common/errors.py

```diff
@@ -26,8 +26,7 @@ def __init__(self, msg, results=None):
         self.results = results
 
     def _to_string(self):
-        return unicode(u"{}, Ansible Results =>\n{}".format(self.message, dump_ansible_results(self.results)))\
-            .encode('ascii', 'backslashreplace')
+        return "{}, Ansible Results =>\n{}".format(self.message, dump_ansible_results(self.results)).encode().decode("utf-8")
```

Comment:
What's the reason for calling `encode()` followed by `decode("utf-8")` here?

Reply:
This approach is used for backward compatibility of code between different versions of Python. Ansible returns different types of dump depending on Python versions.